.github/workflows/CompatHelper.yml

```
@@ -1,19 +1,15 @@
 name: CompatHelper
-
 on:
   schedule:
-    - cron: '00 23 * * *'
-
+    - cron: 0 23 * * *
 jobs:
-  CompatHelper:
+  build:
```

Uh? :slightly_smiling_face: 
```suggestion
  CompatHelper:
```

---

just copying from other projects- this still works I think